webpack/assets/javascripts/react_app/components/BrowserSupport/BrowserSupport.js

```diff
@@ -0,0 +1,55 @@
+import React, { useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { Button } from 'patternfly-react';
+import { sprintf, translate as __ } from '../../common/I18n';
+import { noop } from '../../common/helpers';
+import './BrowserSupport.scss';
+
+const BrowserSupport = ({ show, browserName, initializeBanner }) => {
+  useEffect(() => {
```

Comment:
I think mixing react hooks and `connect` is confusing and affect our unify code and conventions , therefore I prefer to use the class component with lifecycle functions on connected components, until we will embrace react hooks as a convention.

With hooks we can replace `connect` with the new  [redux API](https://react-redux.js.org/next/api/hooks), but until then lets keep our convention